controller/controller.js

```diff
@@ -3,62 +3,76 @@ const mongoose = require('mongoose');
 const bodyParser = require('body-parser');
 
 
-
 //  Add new data
 exports.adddata = async (req, res) => {
 
-    const uname = req.body.name;
-    const uemail = req.body.email;
+    const userName = req.body.name;
+    const userEmail = req.body.email;
+    const userAge = req.body.age;
+    const userBdate = req.body.birthdate;
+    const userPassword = req.body.password;
+
```

Comment:
we can use destructuring here

Reply:
and no need to create new variables we can pass directly to model.create function

Reply:
NOTE: extra variables creation, will need extra memory storage